README.md

```
@@ -19,28 +23,46 @@
 </div>
 
 ---
 
+# The README Project.
+
+<div align="center">
+
+  <em>The Startup & PhD Journey </em><br />
+  <sub><a href="about.md">The journey from the classroom to boardroom </a></sub>
+
+  [Ram Murmu ](https://github.com/rammurmu)
```

markdownlint-cli2 (0.18.1)

Spaces inside link text

(MD039, no-space-in-links)